addons/skin.confluence/language/English/strings.po

```diff
@@ -284,16 +293,17 @@ msgstr ""
 
 #empty string with id 31133
 
+#. In main language strings its used as submenu
 msgctxt "#31134"
-msgid "Home page Videos sub-menu"
+msgid "Home page videos submenu"
```

Comment:
We're using quoted uppercase strings for references, don't we? IMO this would suite here - no strong feelings about it though, was just a thought.